src/qspdlog_model.cpp

```diff
@@ -145,3 +160,39 @@ QVariant QSpdLogModel::headerData(
 
     return QVariant();
 }
+void QSpdLogModel::setForegroundMapping(
+    const std::string& loggerName, std::optional<QBrush> brush
```

Comment:
`const std::string&` -> `std::string_view`
```suggestion
    std::string_view loggerName, std::optional<QBrush> brush
```